### PR TITLE
Fashionable Bomber jacket accessory

### PR DESCRIPTION
**Pull request by Absolutely-Steph**
## About The Pull Request
Basically just the bomber jacket, but as an accessory, by default it gets displayed over hardsuits, so you can now look dashing while flying your fighter jet.

## Why It's Good For The Game
It look good, good looks good.

## Changelog
:cl:
add: Added Thin bomber jackets
/:cl:

### CODE REVIEW

**Comment by Absolutely-Steph** [comment]
there is no way to acquire it right now, i'll add it to the muni vendor later.

**Comment by Absolutely-Steph** [comment]
> Send pics!

ah right, here's it in action on a pilot flight suit, though you can wear it with everything:
unknown

File: nsv13/code/modules/clothing/custom_clothes.dm [hunk 1]
```diff
@@ -249,6 +249,15 @@
 	item_state = "sun"
 	dynamic_hair_suffix = ""
 
+/obj/item/clothing/accessory/bomber_jacket_accessory
+	name = "Thin bomber jacket"
```
Comment (TheFakeElon) [inline nsv13/code/modules/clothing/custom_clothes.dm:253]:
```suggestion
	name = "thin bomber jacket"
```

**Comment by TheFakeElon** [comment]
Clicked on this PR expecting a suicide bomb vest lol

File: nsv13/code/modules/clothing/custom_clothes.dm (hunk 1 above)
Comment (Absolutely-Steph) [reply nsv13/code/modules/clothing/custom_clothes.dm:253]: the underscores make sense, but why make the name lowercase only?

**Comment by Absolutely-Steph** [comment]
Turns out every goddamn item in the game just has random capitalisation, very annoying, I'll never unsee that.
I've added it to the munivendor as a premium item too.
Ready to merge now.

**Comment by covertcorvid** [comment]
Is this added to both accessories files?

**Comment by Absolutely-Steph** [comment]
> Is this added to both accessories files?

yup, ours only stores inhands weirdly enough, I needed to add it to the native bee one in order to have an actual wearable sprite :/

**Review by covertcorvid — APPROVED** [review]

**Comment by Karmic-Skink** [comment]
This is special snowflake as heck.

-1 Disapprove.

File: nsv13/code/modules/clothing/custom_clothes.dm [hunk 2]
```diff
@@ -249,6 +249,15 @@
 	item_state = "sun"
 	dynamic_hair_suffix = ""
 
+/obj/item/clothing/accessory/bomber_jacket_accessory
+	name = "thin bomber jacket"
+	desc = "A cheap, wafer thin replica of a bomber jacket, this would fit on top of most things."
+	icon = 'nsv13/icons/obj/clothing/accessories.dmi'
+	icon_state = "bomberthin"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/clothing/custom_clothes.dm:256]:
```suggestion
	alternate_worn_icon = 'nsv13/icons/mob/custom_accessories.dmi'
	icon = 'nsv13/icons/obj/clothing/accessories.dmi'
	icon_state = "bomberthin"
```
Please do this and then add the worn sprite to a new file in that destination with the right icon state please.

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: nsv13/code/modules/clothing/custom_clothes.dm (hunk 2 above)
Comment (Absolutely-Steph) [reply nsv13/code/modules/clothing/custom_clothes.dm:256]: I can, but this turns it invisible...

**Comment by Karmic-Skink** [comment]
Okay so the actual reasons I don't like this is because
 A: How does a jacket even fit over a space suit (minor gripe)
 B: Why is this specifically that jacket and not including say the ERT jacket (major gripe)

**Comment by Absolutely-Steph** [comment]
> Okay so the actual reasons I don't like this is because A: How does a jacket even fit over a space suit (minor gripe) B: Why is this specifically that jacket and not including say the ERT jacket (major gripe)

-It's a thin, cheaply made replica intended for this
-This entire PR only exists because i saw people talk about wanting bomber jackets over flightsuits, and i thought it'd be interesting/fun to make that.

File: nsv13/code/modules/clothing/custom_clothes.dm (hunk 2 above)
Comment (Bokkiewokkie) [reply nsv13/code/modules/clothing/custom_clothes.dm:256]: Alright nevermind then, I'll note it down to be fixed later.

**Review by Bokkiewokkie — APPROVED** [review]

**Comment by Bokkiewokkie** [comment]
We can seperately add other jackets to be like this later if we really want to

**Pull request merged**